lib/filter/enemy.js

```diff
@@ -99,7 +99,31 @@ exports.update = function (json) {
       }
     }
     */
-
+   
+		console.log( 'Enemy Drops:');
+		var drops = 0;
+		for (enemy in json[round].enemy) {
+			if (json[round].enemy[enemy].children[part].drop_item_list.length > 0) {
+				drops = 1;
+				for (var item in json[round].enemy[enemy].children[part].drop_item_list) {
```

Comment:
I'm a bit confused by both of you on this one lol.

There aren't 2 loops for enemies. There's one loop for each enemy in a round, then there's another loop for each item that an enemy can drop. The problem I had run into before was that sometimes, enemies can drop both gold and an item. If this is the case, it will only display one or the other (and due to the nature of how gold value works, it's usually the gold).

So I had to throw the loop in.

I'll fix the tabbing though, no idea why it went berserk, but hey that's why we don't tab.